### DISCUSSION TITLE
Parasolid import?

### DISCUSSION THREAD

**Discussion by ChrisClems** [assimp/assimp#5607] [Q&A] [answered]
Are parasolid file imports supported? I assumed so because it's a supported export format, but I'm getting an `Error importing file: Failed to determine STL storage representation for...` exception on all of the x_b and x_t files exported from Solid Edge.

Is this an unsupported import format or should I continue troubleshooting?

**Comment by kimkulling** [comment, marked as answer]
This feature is currently not supported.
